aiven/resource_project.go

```diff
@@ -36,46 +42,51 @@ var aivenProjectSchema = map[string]*schema.Schema{
 		Sensitive:   true,
 	},
 	"card_id": {
-		Type:        schema.TypeString,
-		Optional:    true,
-		Description: "Credit card ID",
+		Type:             schema.TypeString,
+		Optional:         true,
+		Description:      "Credit card ID",
+		DiffSuppressFunc: emptyObjectDiffSuppressFunc,
 	},
 	"account_id": {
-		Type:        schema.TypeString,
-		Optional:    true,
-		Description: "Account ID",
-	},
-	"copy_from_project": {
 		Type:             schema.TypeString,
 		Optional:         true,
-		Description:      "Copy properties from another project. Only has effect when a new project is created.",
-		DiffSuppressFunc: createOnlyDiffSuppressFunc,
+		Description:      "Account ID",
+		DiffSuppressFunc: emptyObjectDiffSuppressFunc,
 	},
-	"country_code": {
+	"copy_from_project": {
 		Type:        schema.TypeString,
 		Optional:    true,
-		Description: "Billing country code of the project",
+		Description: "Copy properties from another project. Only has effect when a new project is created.",
```

Comment:
`createOnlyDiffSuppressFunc` was dropped from here. Is that intentional?

Reply:
ups, sorry, wasn't intentional, thx